src/components/ConfirmPopup.js

```diff
@@ -0,0 +1,33 @@
+import Popup from "./Popup.js";
+export default class ConfirmPopup extends Popup {
+  constructor(popupSelector, handleFormSubmit) {
+    super({ popupSelector });
+    this._handleFormSubmit = handleFormSubmit;
+  }
+
+  setEventListeners() {
+    super.setEventListeners();
+    this._popupElement.addEventListener("submit", (evt) => {
+      evt.preventDefault();
+      this._handleFormSubmit(this._card);
+    });
+  }
+
+  open(card) {
+    this._card = card;
+    super.open();
+  }
+
+  close() {
+    super.close();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.